skelebot/systems/execution/commandBuilder.py

```diff
@@ -25,13 +25,23 @@ def build(config, job, args, native=False):
 
     return COMMAND.format(extCommand=EXT_COMMAND[ext], file=job.source, args=arguments, params=params, pre=pre, post=post)
 
-def buildArgs(jobArgs, args, template):
+def buildArgs(argParams, args):
     argString = ""
-    if (jobArgs != None):
-        for arg in jobArgs:
-            value = args[arg.name] if (arg.name in args) else arg.default
-            value = "" if (arg.isBoolean) else value
+    if (argParams != None):
+        for arg in argParams:
+            value = args[arg.name] if (arg.name in args) else None
```

Comment:
This is pretty minor but you can make this more compact (and maybe more efficient) using: `value = args.get(arg.name)`